packages/proxy/test/unit/http/request-middleware.spec.ts

```diff
@@ -11,6 +11,7 @@ describe('http/request-middleware', () => {
     expect(_.keys(RequestMiddleware)).to.have.ordered.members([
       'LogRequest',
       'ExtractIsAUTFrameHeader',
+      'CopyCrossOriginCookies',
```

Comment:
can we add some tests here that verify the attaching of cross origin cookies from the cookie jar to cross origin requests?

Reply:
Added in [`c22269b` (#22320)](https://github.com/cypress-io/cypress/pull/22320/commits/c22269bb0d5bc43a1841b0a56de80b70e68f2550)